pyproject.toml

```diff
@@ -27,4 +27,4 @@ use_parentheses = true
 skip_glob = ["build/*", "stubs/*"]
 # Don't split "import" and "from".
 force_sort_within_sections = true
-known_third_party = ["benchmark_dataset", "datasets", "golden_configs", "models", "numpy", "parameterized", "pytest", "recommonmark", "setuptools", "sklearn", "torch", "torchtext", "torchvision", "utils"]
+known_third_party = ["benchmark_dataset", "datasets", "distutils", "golden_configs", "models", "numpy", "parameterized", "pytest", "recommonmark", "setuptools", "sklearn", "torch", "torchtext", "torchvision", "utils"]
```

Comment:
I think you need to undo this change or the precommit check fails.

Reply:
https://github.com/ruanslv/fairscale/blob/auto_wrap_fix/CONTRIBUTING.md#pre-commit-recommended

These changes came from the pre-commit checks, I'm not managing to do a commit without them. Should I just bypass them? Any idea how to do that?

Reply:
Yeah. Need to run precommit uninstall to bypass it for now.

Reply:
You can also remove the change, run `git commit -m "" --no-verify` to enable pushing to the branch without running a local pre-commit check. There is some mismatch between the local and remove pre-commit checks that i am looking into.

Reply:
Thanks for the command pointer. I've removed the changes to this file.